docs/src/components/Dashboard/Difference.tsx

```diff
@@ -49,8 +49,9 @@ const DataDiffTemplate = ({ location }: PageRendererProps) => {
   `);
 
   const { nodes } = allTracking;
-  const first = nodes.slice(0, 8);
-  const last = nodes.slice(-8);
+  const TRACKED_REPOSITORIES = 11;
```

Comment:
What do you think about naming this variable `TRACKED_REPOSITORIES_COUNT` or similar?